rightscale/cm_inputs.go

```diff
@@ -0,0 +1,28 @@
+package rightscale
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/rightscale/terraform-provider-rightscale/rightscale/rsc"
+)
+
+// cmInputs is a helper function that returns fields representing valid
+// RightScale rcl input parameters built from the resource data "inputs"
+// field.  cmInputs keys are SO_ANGRY_AND_SHOUTING, so upcase for success.
+func cmInputs(f []interface{}) (rsc.Fields, error) {
+	var inputs rsc.Fields
+	mapify := make(map[string]string)
+	for _, i := range f {
+		v, ok := i.(map[string]interface{})
+		if !ok {
+			return inputs, fmt.Errorf("inputsList does not appear to be properly handled as a string: %v", ok)
+		}
+		for k, v2 := range v {
+			upK := strings.ToUpper(k)
```

Comment:
Should we really be doing that? I don't think RS actually enforces that inputs be `UPPER_CASE` although that's certainly the convention. The provider should probably not muck with whatever names are provided by the user in the TF definition.

Reply:
Hummmm - for some reason I thought caps was mandated.  Ok, I'll remove that logic.